emf/loadflow_tool/helper.py

```diff
@@ -309,4 +309,20 @@ def get_metadata_from_filename(file_name):
             file_metadata["Model.domain"],\
             file_metadata["Model.forEntity"] = entity_and_area_list
 
-    return file_metadata
\ No newline at end of file
+    return file_metadata
+
+
+def export_model(network: pypowsybl.network, CGM_meta):
+
+    bytes_object = network.save_to_binary_buffer(
+        format="CGMES",
+        parameters={
+            "iidm.export.cgmes.modeling-authority-set": CGM_meta['pmd:modelingAuthoritySet'],
+            "iidm.export.cgmes.base-name": filename_from_metadata(CGM_meta).split(".xml")[0],
+            "iidm.export.cgmes.profiles": "SV,SSH,TP,EQ",
+            "iidm.export.cgmes.naming-strategy": "cgmes",  # identity, cgmes, cgmes-fix-all-invalid-ids
```

Comment:
Investigate the difference between "cgmes" and "cgmes-fix-all-invalid-ids"

Reply:
![image](https://github.com/Baltic-RCC/EMF/assets/91978404/db6da61b-ccfa-4880-85ff-54ddd0247ab6)

Reply:
we should change it to the fix-all-invalid-ids, to reduce possible errors at OPDE publication